sdk/cosmos/azure-cosmos-table/pom.xml

```diff
@@ -0,0 +1,23 @@
+
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-cosmos-table</artifactId>
+  <packaging>pom</packaging>
+  <version>1.0.0</version><!-- Need not change for every release-->
+  <modules>
+    <module>azure-cosmos-table</module>
+  </modules>
+
+  <dependencies>
+  <dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-core</artifactId>
+    <version>1.5.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+  </dependency>
+  </dependencies>
+</project>
```

Comment:
You're probably going to want to add a similar [plugin configuration section](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/pom.xml#L271) for maven-enforcer-plugin which, at least, will have the com.azure:* bannedDependency include. Keep in mind that anything that's not com.azure:* will need to get architect approval if it's not already one of the approved includes.